src/AngularWrapper.jsx

```diff
@@ -0,0 +1,117 @@
+import React, { Component, createElement } from 'react';
+import PropTypes from 'prop-types';
+import angular from 'angular';
+import AngularTemplate, { ensureScopeAvailable, reactAngularModule } from './angularTemplate';
+import camelCaseToDash from './utils/camelCaseToDash';
+
+class AngularWrapper extends Component {
+  state = {
+    angularActive: false,
+  };
+
+  component = null;
```

Comment:
Mixture of component & angularComponent is very confusing a.t.m. Maybe clear up naming.